CONTRIBUTING.md

```diff
@@ -91,25 +91,73 @@ it will create a new folder inside `packages` with a simple boilerplate for your
 
 ### Prerelease
 
-- Every merged PR should trigger an automatic canary release
-- The version number is calculated by a combination of the output of `git describe` and the `publish:prerelease` script in the root package.json. Specifically having `major` is appropriate for the pre-7.0 development phase, but we should change it to `minor` after 7.0.0 is finally released.
+A [prerelease](https://github.com/Turfjs/turf/blob/master/.github/workflows/prerelease.yml) action is available that publishes a canary release for every commit or PR merged to the master branch.  However, this action is only [enabled](https://github.com/Turfjs/turf/actions/workflows/prerelease.yml) when needed.
+
+When used, it publishes an alpha release to NPM (e.g. `7.0.0-alpha.116` where 116 is the number of commits to master since the last release tag).
+- The version number is calculated by a combination of the output of `git describe` and the `publish:prerelease` script in the root package.json. It is typically setup to do a `minor` prerelease, but can be changed, such as prior to a `major` release.
 
 ### Release
-- If necessary, make and merge a PR to contain any last minute changelog entries or other housekeeping items
-- Make a PR that contains the file changes from `pnpm lerna version --no-push 7.0.0` (update the version as necessary)
-- When pushing the release PR, use `--follow-tags` to make sure the relevant tag is also pushed
-- The CI will start doing the release when the tag is created
-- Don't forget to merge your PR
+
+A turf release is done from your local computer using a clean, up-to-date version of the turf master branch.
+
+- If necessary, make and merge a PR with any last minute housekeeping items.
+  - Turf's documentation (README.md files) should be up to date as they are updated automatically on commit, based on the JSDoc comments in the package files.  See the [documentation](#documentation) for how to regenerate them if needed.
+- Review code commits and decide the new version number to be published (for example 7.0.1 or 7.1.0).
+  - If there are breaking changes, then it should be a major version bump, e.g. 7.x.x to 8.0.0.  This project follows [semantic versioning](https://semver.org/).
+- When running release commands below, replace `7.0.0` with the version number you are releasing.  Now, let's get started.
+
+- fetch the latest code from remote origin
+```bash
+git fetch origin
+```
+
+- create a release branch, replace mf with your initials to make it clear whose branch it is.
+```bash
+git checkout origin/master -b mf/release-7.0.0
+```
+
+- increment the version number of all packages, without pushing to origin.  This will also create a release tag.
+```bash
+pnpm lerna version --no-push 7.0.0
```

Comment:
Should this include `--no-commit-hooks` as well? Otherwise we might accidentally generate readmes, etc. Ideally it shouldn't happen, but if something goes awry the release PR would end up containing things other than packages/*/package.json and lerna.json

Reply:
Added, seems logical.  Testing needed.